releasenotes.md

```diff
@@ -1,8 +1,27 @@
-- [v0.5.0](#v0.5.0)
-- [v0.4.0](#v0.4.0)
-- [v0.3.0](#v0.3.0)
-- [v0.2.1](#v0.2.1)
-- [v0.2.0](#v0.2.0)
+- [v0.6.0](#v060)
+- [v0.5.0](#v050)
+- [v0.4.0](#v040)
+- [v0.3.0](#v030)
+- [v0.2.1](#v021)
+- [v0.2.0](#v020)
+
+## v0.6.0
+
+### Enhancement
+* Add a way to check if the dns is ready (#81)
+* ci: add hyperkube 1.11 (#77)
```

Comment:
This is not an enhancement in term of pupernetes software.
It should go to the *other* section.